test/unit/models/sitewide_setting_test.rb

```diff
@@ -0,0 +1,12 @@
+require "test_helper"
+
+class SitewideSettingTest < ActiveSupport::TestCase
+ 
+  test "toggling reshuffle mode republished the ministers index page" do
+    payload = PublishingApi::MinistersIndexPresenter.new
+    
+    Services.publishing_api.expects(:publish).with(payload.content_id, nil, locale: "en").once
```

Comment:
I'd suggest using the [stub methods here](https://github.com/alphagov/gds-api-adapters/blob/8c5224f9c331cf6a22d6065ef7f5de9fcbbbed20/lib/gds_api/test_helpers/publishing_api.rb#L71-L84) instead of `expects` because it means we're mocking at a "high level", i.e. the HTTP level, so we have more confidence in the code.

They're used in a similar way and here's an example: https://github.com/alphagov/whitehall/blob/da56426b9ae1a8b430185bf3f6da26fc4d0e68af/test/unit/whitehall/publishing_api_test.rb#L25

Reply:
👍